application/views/account/select_subscription.php

```diff
@@ -6,205 +6,222 @@
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>Les 5 marques les moins ajoutées au panier par utilisateurs</td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td colspan="4" class="sep">Entreprises</td>
-            </tr>
-            <tr>
-                <td>Combien d'utilisateurs visitent mon magasin</td>
-                <td><span class="tick">&#10004;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>Combien de produits le magasin le plus visité met en ligne chaque semaine</td>
-                <td><span class="tick">&#10004;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>À quelle distance se trouvent les utilisateurs qui visitent mon magasin </td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>Combien d'utilisateurs m'ont choisi comme magasin préféré</td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>À quelle distance se trouvent les utilisateurs qui ajoutent mes produits à leur panier</td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>Quel est le magasin le plus visité</td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-            <tr>
-                <td>Quel est le magasin qui a plus de produits dans la liste des produits moins cher</td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tickred">&#x2716;</span></td>
-                <td><span class="tick">&#10004;</span></td>
-            </tr>
-                <tr><td class="mask"></td>
-                <td><md-button class='md-raised md-primary md-hue-2' ng-click="ctrl.selectSubscription(1)">Souscrire</md-button></td>
-                <td>
-                    <div id="dropin-container"></div>
```

Comment:
Please make sure the drop-in control for the visa card payments are still available.

Reply:
Yes